documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -770,68 +768,109 @@ where:
 
 .. note::
 
-   Like ``line_proj``, ``sphere_proj`` can be `defined using
-   positional or keyword arguments <#poskwarg>`_. However the same
+   Like ``line_proj``, ``sphere_proj`` can be :ref:`defined using
+   positional or keyword arguments <poskwarg>`. However the same
    recommendation applies that positionals should only be used to
-   define ``ez`` and ``ex``.
+   define ``u`` and ``v``.
 
 ``sphere_proj`` defines a spherical coordinate system, where:
 
 * |Q|
 
-  is the radius from the origin (``offset``) in :math:`hkl`
+  The radius from the origin (``offset``) in :math:`hkl`
 
 * :math:`\theta`
 
-  is the angle measured from :math:`e_z` to the vector (:math:`\vec{q}`),
-  i.e. :math:`0^{\circ}` is parallel to :math:`e_z` and :math:`90^{\circ}` is
-  perpendicular to :math:`e_z`
+  The angle measured from :math:`u` to the vector (:math:`\vec{q}`),
+  i.e. :math:`0^{\circ}` is parallel to :math:`u` and :math:`90^{\circ}` is
```

Comment:
```suggestion
 so that a \theta value of :math:`0^{\circ}` means that `q` is parallel to :math:`u`; and a \theta value of :math:`90^{\circ}`means that `q` is
```